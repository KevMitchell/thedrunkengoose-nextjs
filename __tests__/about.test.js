import { render } from '@testing-library/react'
import About from '../pages/about'

test('renders navigation', () => {
  const { container } = render(<About />)
  expect(container).toMatchSnapshot()
})
