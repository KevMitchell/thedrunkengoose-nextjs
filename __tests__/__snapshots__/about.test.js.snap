// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders navigation 1`] = `
<div>
  <div
    class="jsx-1463981424 about"
  >
    <div
      class="jsx-1463981424 text"
    >
      <div
        class="jsx-1463981424"
      >
        The Drunken Goose is a cocktail bar serving the very best in drinks and company.
      </div>
      <div
        class="jsx-1463981424"
      >
        We can be found in LARP events around the UK; all are welcome to come sit by the fire, drink, eat, gossip, and sing.
      </div>
      <div
        class="jsx-1463981424"
      >
        Private tables are available on request.
      </div>
    </div>
    <img
      alt="Drunken Goose Crew"
      class="jsx-1463981424"
      src="./img/crew1.jpg"
    />
    <img
      alt="Drunken Goose"
      class="jsx-1463981424"
      src="./img/goose1.jpg"
    />
    <div
      class="jsx-1463981424 text"
    >
      <div
        class="jsx-1463981424"
      >
        <a
          alt="Profound Decisions: Empire"
          class="jsx-1463981424"
          href="https://www.profounddecisions.co.uk/empire"
        >
          Profound Decisions: Empire
        </a>
      </div>
      <div
        class="jsx-1463981424"
      >
        The staff of The Drunken Goose are part of the Nation of Navarr at Empire events. The Goose can be found in the Navarr woods at the intersection between the main camp path and Murder Alley leading to the Wintermark camp.
      </div>
    </div>
    <img
      alt="Drunken Goose Crew"
      class="jsx-1463981424"
      src="./img/crew2.jpg"
    />
    <img
      alt="Drunken Goose Crew"
      class="jsx-1463981424"
      src="./img/goose2.jpg"
    />
    <img
      alt="Drunken Goose Crew"
      class="jsx-1463981424"
      src="./img/goose3.jpg"
    />
    <div
      class="jsx-1463981424 text last"
    >
      <div
        class="jsx-1463981424"
      >
        For queries we can be contacted at 
        <a
          class="jsx-1463981424"
          href="mailto:contact@thedrunkengoose.com"
        >
          contact@thedrunkengoose.com
        </a>
      </div>
    </div>
  </div>
</div>
`;
